zcash_client_sqlite/src/wallet/init/migrations/fix_bad_change_flagging.rs

```diff
@@ -5,12 +5,11 @@ use std::collections::HashSet;
 use rusqlite::named_params;
 use schemerz_rusqlite::RusqliteMigration;
 use uuid::Uuid;
-use zip32::Scope;
 
 use crate::{
     wallet::{
         init::{migrations::fix_broken_commitment_trees, WalletMigrationError},
-        scope_code,
+        KeyScope,
```

Comment:
```suggestion
        scope_code,
```
(or add `zip32::Scope::encode`).

Reply:
Resolved via 0dcb2b2e74f6d8f0552c6cab01e2e0061150b3ce instead.